src/app/api/events/id/route.ts

```diff
@@ -37,10 +36,13 @@ export default async function handler(
         imageUrl: event.image.baseUrl,
         dateTime: convertDate(event.dateTime),
       };
-      return res.status(200).json(result);
+      return NextResponse.json(result, { status: 200 });
     }
```

Comment:
I didn't have idea of utilizing `NextResponse` instead of `res.status()`, and that looks cool! 😎

Reply:
To be honest, I just mimic this usage: https://nextjs.org/docs/app/building-your-application/routing/route-handlers#caching. We can use `Response.json()` once upgrading Typescript to 5.2.0 or higher.